.gitignore

```
@@ -1,5 +1,7 @@
 .DS_Store
 /.build
 /Packages
-/*.xcodeproj
 xcuserdata/
+
+**/docsets/*
+.docsGeneratedByLint/
```

A new line at file end should be added

```suggestion
.docsGeneratedByLint/

```